sci-mathematics/pymc/pymc-2.3.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
```

Comment:
I checked on PyPI and Github, and the authors say they support python 2.7, 3.4 and 3.5, so add those implementations too.